opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/OnConfigurationChangedListener.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+ */
+package org.opengrok.indexer.configuration;
+
+public interface OnConfigurationChangedListener {
```

Comment:
I think the name should be `ConfigurationChangedListener`

Reply:
https://developer.android.com/reference/android/view/View.OnClickListener

It's a matter of taste but the naming like this is used quite often.

Reply:
```
opengrok-web/src/main/java/org/opengrok/web/WebappListener.java
opengrok-indexer/src/main/java/org/opengrok/indexer/index/DefaultIndexChangedListener.java 
opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/NonSymbolMatchedListener.java
opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/SymbolMatchedListener.java
opengrok-indexer/src/main/java/org/opengrok/indexer/index/IndexChangedListener.java
```